### PR TITLE
fix(Account Statement): misc UI updates

### PR DESCRIPTION
**Pull request by jniles**
This commit fixes the following issues on the Account Statement module:
 1. Unused requires are removed
 2. The "Configure" cogs are now "Columns" like on other modules
 3. The "print" labels are now "Export to PDF"
 4. Debits/Credits now sort numerically instead of as text

### CODE REVIEW

File: client/src/modules/account_statement/account_statement.html [hunk 1]
```diff
@@ -16,17 +16,17 @@
           <ul uib-dropdown-menu role="menu" class="dropdown-menu-right">
             <li role="menuitem">
               <a href data-method="configure" ng-click="AccountStateCtrl.openColumnConfigModal()">
-                <i class="fa fa-cogs"></i> <span translate>FORM.BUTTONS.CONFIGURE</span>
+                <i class="fa fa-columns"></i> <span translate>FORM.LABELS.COLUMNS</span>
               </a>
             </li>
             <li role="separator" class="divider"></li>
             <li role="menuitem">
-              <a href data-method="print" id="print" ng-click="AccountStateCtrl.exportPdf()">
```
Comment (mbayopanda) [inline client/src/modules/account_statement/account_statement.html:24]: Good catch

File: client/src/modules/account_statement/account_statement.html [hunk 2]
```diff
@@ -16,17 +16,17 @@
...
+              <a href data-method="export-pdf" id="export-pdf" ng-click="AccountStateCtrl.exportPdf()">
```
Comment (mbayopanda) [inline client/src/modules/account_statement/account_statement.html:24]: 👍 meaningful name

File: client/src/modules/account_statement/account_statement.ctrl.js [hunk 3]
```diff
@@ -74,6 +73,7 @@ function AccountStatementController(GeneralLedger, Notify, Journal,
 
     {
       field            : 'debit_equiv',
+      type : 'number',
```
Comment (mbayopanda) [inline client/src/modules/account_statement/account_statement.ctrl.js:76]: 👍

**Review by mbayopanda — APPROVED** [review]

**Comment by jniles** [comment]
@mbayopanda, can you "bors r+" it?

**Pull request merged**